src/lib/driver/message_handler.ts

```diff
@@ -65,6 +65,24 @@ export class DriverMessageHandler {
       case DriverCommand.enableErrorReporting:
         driver.enableErrorReporting();
         return {};
+      case DriverCommand.softReset:
+        if (driver.controller.isAnyOTAFirmwareUpdateInProgress()) {
+          throw new UnableToResetError("soft");
+        }
+        await driver.softReset();
+        return {};
+      case DriverCommand.trySoftReset:
+        if (driver.controller.isAnyOTAFirmwareUpdateInProgress()) {
+          throw new UnableToResetError("soft");
+        }
+        await driver.trySoftReset();
```

Comment:
Is this actually a driver function, why ?

Reply:
yes to the first question, @AlCalzone for the second

Reply:
Error reports are sent to Sentry, so this is a way to opt-in.